utils/lj-releng

```diff
@@ -0,0 +1,213 @@
+            print("$file: $orig_ver ($ver)\n");
+            last;
+
+        } elsif (/(?x) (?:_VERSION|version) \s* = \s* ([a-zA-Z_]\S*)/) {
+            print("$file: $1\n");
+            $found_ver = 1;
+            last;
+        }
+
+        if ($ver and $version and !$skipping) {
+            if ($version ne $ver) {
+                die "$file: $ver != $version\n";
+            }
+        } elsif ($ver and !$version) {
+            $version = $ver;
+        }
+    }
+    # if (!$found_ver) {
+    #     w("WARNING: No \"_VERSION\" or \"version\" field found in `$file`.");
+    # }
+    close $in;
+
+    #p("Checking use of Lua global variables in file $file...");
+    #p("op no. line     opcode  args        ; code");
+    my $cmd = "luajit -bL $file";
+    open $in, "$cmd|"
+        or die "cannot open output pipe for \"$cmd\": $!";
+    my @sections;
+    my $sec;
+    while (<$in>) {
+
+        #warn "line: $_";
+
+        if (/^-- BYTECODE -- \S.*?:(\d+)-\d+$/) {
+            my $def_line = $1;
+            #warn "$file: $line";
+            if (defined $sec) {
+                push @sections, $sec;
+
+            }
+            $sec = {
+                def_line => $def_line,
+                gsets => [],
+                ggets => [],
+            };
+            next;
+        }
+
+        if (/^ \d+ \s+ \[ (\d+) \] \s+ (?: \W+ \s+ )? G([GS])ET \s+ .*? ; \s+ \"([^"]+)" $/x) {
+            my ($line, $op, $name) = ($1, $2, $3);
+
+            #warn "found: $line $op $name";
+            if ($op eq 'S') {
+                push @{ $sec->{gsets} }, [$line, $name];
+            } else {
+                push @{ $sec->{ggets} }, [$line, $name];
+            }
+
+            next;
+        }
+
+        if (/^ \d+ \s+ \[ \d+ \] \s+ (G[GS]ET) \s+ $/x) {
+            die "bad $1 instruction: $_";
+        }
+    }
+    close $in;
+
+    if (defined $sec) {
+        push @sections, $sec;
+    }
+
+    my $last_idx = $#sections;
+    my $i = 0;
+    for my $sec (@sections) {
+        my $def_line = $sec->{def_line};
+
+        my $gsets = $sec->{gsets};
+        my $ggets = $sec->{ggets};
+
+        for my $gset (@$gsets) {
+            $hits++;
+            my ($line, $name) = @$gset;
+            warn "${RED}ERROR${NC}: $file: line $line: setting the Lua global ",
+                 "\"$name\"\n";
+        }
+
+        if ($i == $last_idx) {
+            # being the top-level chunk
+
+            for my $gget (@$ggets) {
+                my ($line, $name) = @$gget;
+
+                if ($name =~ /^ (?: require|type|tostring|error|ngx|ndk|jit
+                                   |setmetatable|getmetatable|string|table|io
+                                   |os|print|tonumber|math|pcall|xpcall|unpack
+                                   |pairs|ipairs|assert|module|package
+                                   |coroutine|[gs]etfenv|next|rawget|rawset
+                                   |loadstring|dofile|collectgarbage|load
```

Comment:
`load` is added here which is not present in original lj-relang causing lint check to fail